src/build/env.py

```diff
@@ -70,6 +75,7 @@ def _get_env_path(self, path):  # type: (str) -> Optional[str]
         return sysconfig.get_path(path, vars=self._env_vars)
 
     def _symlink_relative(self, path):  # type: (Optional[str]) -> None
+        assert self._path
```

Comment:
Why not raise an error? Assertions can be disabled, raising an error cannot?

Reply:
Right, sorry! Asserts shouldn't be used here.